packages/core/src/metrics/exports-default.ts

```diff
@@ -39,20 +38,9 @@ function gauge(name: string, value: number, data?: MetricData): void {
   metricsCore.gauge(MetricsAggregator, name, value, data);
 }
 
-/**
- * Returns the metrics aggregator for a given client.
- */
-function getMetricsAggregatorForClient(client: Client): MetricsAggregatorInterface {
-  return metricsCore.getMetricsAggregatorForClient(client, MetricsAggregator);
-}
-
-export const metricsDefault = {
+export const metricsDefault: Metrics = {
   increment,
   distribution,
   set,
   gauge,
-  /**
-   * @ignore This is for internal use only.
-   */
-  getMetricsAggregatorForClient,
```

Comment:
@AbhiPrasad This is not used anywhere, is there a reason to re-export this here? I figured this is safe to remove here...?

Reply:
It's used by electron: https://github.com/getsentry/sentry-electron/blob/2449a1c05abd28abfe6eeb6bdedde0f2b9d92666/src/main/ipc.ts#L135

Reply:
We should leave a comment about this

Reply:
I can leave it in then, probably better then! 👍